runtime/gc_realtime/RealtimeMarkTask.cpp

```diff
@@ -47,9 +46,7 @@ MM_RealtimeMarkTask::setup(MM_EnvironmentBase *envBase)
 	MM_EnvironmentRealtime *env = MM_EnvironmentRealtime::getEnvironment(envBase);
 	MM_GCExtensions *extensions = MM_GCExtensions::getExtensions(env);
 
-	env->_markStats.clear();
-	env->getGCEnvironment()->_markJavaStats.clear();
-	env->_workPacketStats.clear();
+	envRealtime->getExtensions()->realtimeGC->getRealtimeDelegate()->clearGCStatsEnvironment(env);
```

Comment:
instead of `envRealtime->getExtensions()` you can use `extensions` declared line above